src/20250421_Multi_GHOGnosisLaunch/utils/GHOLaunchConstants.sol

```diff
@@ -0,0 +1,84 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+/**
+ * @title GHOLaunchConstants
+ * @notice Library containing all constants used across the GHO Gnosis Launch proposal
+ */
+library GHOLaunchConstants {
+  // Common Addresses
+  address internal constant RISK_COUNCIL = 0x8513e6F37dBc52De87b166980Fa3F50639694B60;
+
+  // CCIP Rate Limits
+  uint128 internal constant CCIP_RATE_LIMIT_CAPACITY = 1_000_000e18;
+  uint128 internal constant CCIP_RATE_LIMIT_REFILL_RATE = 200e18;
+  uint128 internal constant CCIP_BUCKET_CAPACITY = 15_000_000e18;
+
+  // Arbitrum Addresses
```

Comment:
better to use the addresses from address book here (the ones that are there): https://github.com/bgd-labs/aave-address-book/blob/main/src/GhoArbitrum.sol#L10

same for other networks